### PR TITLE
Change WSO2 Inc. to LLC

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Change WSO2 Inc. to LLC.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: public/0.990/learn/api-docs/ballerina/docerina-theme/js/theme.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2018, WSO2 LLC (http://www.wso2.org) All Rights Reserved.
```
Comment (keizer619) [inline public/0.990/learn/api-docs/ballerina/docerina-theme/js/theme.js:2]:
```suggestion
 * Copyright (c) 2022, WSO2 LLC (http://www.wso2.org) All Rights Reserved.
```

File: public/spec/lang/2020R1/lib/boolean.bal [hunk 2]
```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2020 WSO2 LLC (http://www.wso2.org) All Rights Reserved.
```
Comment (keizer619) [inline public/spec/lang/2020R1/lib/boolean.bal:1]:
```suggestion
// Copyright (c) 2022 WSO2 LLC (http://www.wso2.org) All Rights Reserved.
```

File: public/spec/lang/2020R1/lib/array.bal [hunk 3]
```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2019, 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2019, 2020 WSO2 LLC (http://www.wso2.org) All Rights Reserved.
```
Comment (keizer619) [inline public/spec/lang/2020R1/lib/array.bal:1]: This has to be changed for 2022 License

Comment (praneesha) [reply public/spec/lang/2020R1/lib/array.bal]: Updated.

File: public/spec/lang/2019R2/lib/stream.bal [hunk 4]
```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2018, 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2018, 2019 WSO2 LLC (http://www.wso2.org) All Rights Reserved.
```
Comment (keizer619) [inline public/spec/lang/2019R2/lib/stream.bal:1]:
```suggestion
// Copyright (c) 2022 WSO2 LLC (http://www.wso2.org) All Rights Reserved.
```

**Review by keizer619 — APPROVED** [review]

**Pull request merged**